internal/cluster/nodes/tracker_static.go

```diff
@@ -0,0 +1,19 @@
+package nodes
+
+import (
+	"context"
+
+	"github.com/atlassian/gostatsd"
+)
+
+// staticNodeTracker is a tracker with a static list of hosts
+type staticNodeTracker struct{}
+
+func NewStaticNodeTracker(picker consistentNodePicker, host []string) gostatsd.Runnable {
```

Comment:
Technically this should be returning a `NodeTracker`, but it's unused at present.  Awkward.

Reply:
Do you have a fix for this in future?

Reply:
Not originally, but I will.  In the original draft I just returned a runnable, but since realized I'm probably going to extend the interface, so it's better to give myself the option now.